src/components/DataTable/DataTable.vue

```diff
@@ -79,11 +105,64 @@ export default defineComponent<Props>({
       dispatchChangeEvent({ type: 'SEARCH_UPDATE', payload: searchQuery });
     }
 
+    function handlePageNumberChange(pageNumber: number): void {
+      context.emit('change:page-number', pageNumber);
+      dispatchChangeEvent({ type: 'PAGE_NUMBER_UPDATE', payload: pageNumber });
+    }
+
+    function handlePageSizeChange(pageSize: number): void {
+      context.emit('change:page-size', pageSize);
+      dispatchChangeEvent({ type: 'PAGE_SIZE_UPDATE', payload: pageSize });
+    }
+
     return {
       handleSearchChange,
+      handlePageNumberChange,
+      handlePageSizeChange,
     };
   },
 });
 </script>
 
-<style scoped lang="scss"></style>
+<style>
+.sb-show-main {
+  padding: 20px !important;
+}
+</style>
+
+<style scoped lang="scss">
+.data-table {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.table-container {
+  height: 100%;
+  margin-bottom: 0;
+  flex: 1;
+
+  ::v-deep table tr:last-child td {
+    border-bottom: 0;
+  }
```

Comment:
is it required? do we have some other way?

Reply:
Yes, it is required. There is no other way at the moment.